### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by fabxc**
This removes some things that no longer apply and syncs the installation intructions with the Prometheus repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,29 +4,28 @@
...
+of the GitHub repository. Using the latest production release binary
+is the recommended way of installing Alertmanager.
 
-    sudo apt-get install build-essential libc6-dev
+### Container images
 
-Red Hat family:
-
-    sudo yum install glibc-static
+Container images are available on [Quay.io](https://quay.io/repository/prometheus/alertmanager).
```
Comment (grobie) [inline README.md]: I think @sdurrheimer has a point, "container images" is not very specific. It'd be helpful to know that this is about docker images.

File: README.md [hunk 2]
```diff
@@ -4,29 +4,28 @@
...
 
-### Build dependencies
+There are various ways of installing Alertmanager.
 
-These dependencies are necessary for building Alertmanager. There are no runtime dependencies, as the resulting binary is statically linked.
+### Precompiled binaries
 
-Debian family:
+Precompiled binaries for released versions are available in the
+[*releases* section](https://github.com/prometheus/alertmanager/releases)
```
Comment (fabxc) [inline README.md]: Ah, this was just copying the pattern from the Prometheus README. This is from before we had your release section. Will update on both sides.

File: README.md (hunk 1 above)
Comment (brancz) [inline README.md]: I agree, if we're some day going to add appc/oci images then we can still rename this

**Comment by fabxc** [comment]
Both addressed.

**Comment by brancz** [comment]
lgtm

**Pull request merged**